test/testmachinery/shoots/vpntunnel/vpntunnel.go

```diff
@@ -93,7 +93,7 @@ var _ = ginkgo.Describe("Shoot vpn tunnel testing", func() {
 					pod.Namespace,
 					pod.Name,
 					"pause",
-					fmt.Sprintf("curl -k -v -XGET  -H \"Accept: application/json, */*\" -H \"Authorization: Bearer %s\" \"%s/api/v1/namespaces/%s/pods/%s/log?container=logger\"", token, f.Shoot.Status.AdvertisedAddresses[0].URL, pod.Namespace, pod.Name),
```

Comment:
Is there a reason `-XGET` got removed?

Reply:
Good question! When I was debugging the test, I was the warning:

```
Note: Unnecessary use of -X or --request, GET is already inferred.
```

That's why I removed it.